packages/p2p-v2/src/pages/buy-sell/screens/BuySellTable/BuySellTable.scss

```diff
@@ -0,0 +1,32 @@
+.p2p-v2-buy-sell-table {
+    & .p2p-v2-table {
+        &__header {
+            padding: 1.6rem;
+            border-bottom: 1px solid #f2f3f4;
+            grid-template-columns: 2fr 1.4fr 1.4fr 2.4fr 0.8fr;
+        }
+        &__content {
+            overflow: auto;
+
+            @include mobile {
+                // stylelint-disable-next-line declaration-no-important
+                height: calc(100vh - 21rem) !important;
+            }
+        }
+    }
+
+    &__tabs {
+        & .derivs-primary-tabs {
```

Comment:
isn't this the same as the one in buysellheader.scss?

Reply:
theres only a slight change which is the margin 😅